frontend/packages/console-shared/src/components/editor/YAMLEditorToolbar.scss

```diff
@@ -5,7 +5,7 @@
     justify-content: flex-end;
     margin: 2px;
     z-index: 1;
-    @media(max-width: 767px) {
+    @media(max-width: 991px) {
```

Comment:
```suggestion
    @media(max-width: $screen-md-max) {
```

Reply:
The original variable was `$screen-sm-max` which is `991px`. The md one is `1199px` which we don't want. 
But the main reason I added `991px` directly and didn't use the variable is because to be able to use the variable I have to add `@import '~bootstrap-sass/assets/stylesheets/bootstrap/variables';` in the scss file and I was not sure how that would affect the bundle size.

Reply:
Updated.